docs/sdk/push-notifications/fragments/ios/getting-started.md

```diff
@@ -1,105 +1,170 @@
 Enable your users to receive mobile push messages sent from the Apple (APNs) and Google (FCM/GCM) platforms. The CLI deploys your push notification backend using [Amazon Pinpoint](http://docs.aws.amazon.com/pinpoint/latest/developerguide/).
 You can also create Amazon Pinpoint campaigns that tie user behavior to push or other forms of messaging.
 
-## Set Up Your Backend
-
-1. Complete the [Get Started](./start) steps before you proceed.
-
-2. Complete the [Setting Up APNS Guide](./push-notifications-setup-apns). This will guide you through the process of setting up an App ID, SSL certificate, provisioning profile, entitlements, distribution certificate, and provisioning profile, which are required to deliver notifications to iOS devices.
+## Pre-requisite
 
-3. Use the CLI to add notifications to your cloud-enabled backend and app.
+- Enrolled in the Apple Developer Program
+- Actual device to run the app on
 
-    In a terminal window, navigate to your project folder (the folder that typically contains your project level `xcodeproj` file), and add the SDK to your app.
-
-    ```bash
-    $ cd ./YOUR_PROJECT_FOLDER
-    $ amplify add notifications
-    ```
+1. Complete the [Get Started](./start) steps before you proceed.
```

Comment:
is this the correct link for the new site path? For the root getting started for iOS SDK should be `/start?integration=ios`

Reply:
Correct link notation should be "~/<path>/<file>.md#<section>" no need for ?platform etc.

Reply:
i think you can also do relative paths like "./sdk/push-notifications/fragments/ios/setup-apns". not sure because when i run the server locally, the links are still not working.

Reply:
we should fix all of these: https://github.com/aws-amplify/docs/issues/1454